libs/environment/src/utils/validate-configuration.ts

```diff
@@ -3,3 +3,7 @@ import z from 'zod';
 export const configSchema = z.object({
```

Comment:
chore (blocking): Can we just remove the configSchema now? Its kind of pointless as we are validating the structure of the object we literally just created on line 176

Reply:
we still have old alternative working. I thought leaving old validation for this branch would be still useful. Maybe removal of it should be make along with this old alternative, just after deploying and changing envs on netlify for save consistent transition on new configuration source?

Reply:
I see what you are saying. Personally I think we should just make a clean switch to _only_ using the toml files rather than take on this debt. Every app has VEGA_CONFIG_URLs configured by .env files so lets lets just update those for each app in this PR. Thoughts @dexturr ?

Reply:
As long as we don't remove the static files as they are currently, I'm all for simplicity. The live token site is using those, but that is off the main branch which is super diverged from this one and will be a clean release once the next mainnet release goes out. cc @notbot00

Reply:
Okay yeah lets not touch them. @macqbat can you update this PR so that we only ever fetch from the toml file in github then we can remove the url matching condition keeping things simpler. We'll also need to update all the .env files to point to the github file

Reply:
done